example/src/Views/Checkout/PaymentMethodsView.js

```diff
@@ -63,4 +50,22 @@ const PaymentMethods = () => {
   );
 };
 
+const PaymentMethodButton = (props) => {
+  const { onPress, title, icon } = props;
```

Comment:
```suggestion
const PaymentMethodButton = ({ onPress, title, icon }) => {
```

Reply:
Fixed in [6eccd53](https://github.com/Adyen/adyen-react-native/pull/208/commits/6eccd532a749f4fd71fb7ea00bce9de13b384892)